### ISSUE TITLE
Cannot find module 'qunit'

### ISSUE THREAD

**Issue by plannigan** [Kotlin/kotlin-frontend-plugin#53]
I was trying to run the tests in a project I was starting, but they were failing with

> Cannot find module 'qunit'

I assumed I must be doing something wrong, so I tried the example project and got the same result.

Does the plugin assume qunit is already installed somewhere on the development machine? The readme doesn't mention qunit outside of the karma configuration. The frontend-only example doesn't even have that. It's not clear if this is a bug in the plugin or lack of proper documentation/up-to-date example projects.

Output from running from the example directory in a clean checkout of  the repository at e8aec4d.
```
kotlin-frontend-plugin/examples/frontend-only$ ./gradlew test

> Task :karma-config UP-TO-DATE
A problem was found with the configuration of task ':karma-config'. Registering invalid inputs and outputs via TaskInputs and TaskOutputs methods has been deprecated and is scheduled to be removed in Gradle 5.0.
 - Directory 'kotlin-frontend-plugin/examples/frontend-only/karma.config.d' specified for property '$1' does not exist.

> Task :karma-run-single FAILED
module.js:545
    throw err;
    ^

Error: Cannot find module 'qunit'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.resolve (internal/module.js:18:19)
    at initQUnit (kotlin-frontend-plugin/examples/frontend-only/build/node_modules/karma-qunit/lib/index.js:14:39)
    at Array.invoke (kotlin-frontend-plugin/examples/frontend-only/build/node_modules/di/lib/injector.js:75:15)
    at Injector.get (kotlin-frontend-plugin/examples/frontend-only/build/node_modules/di/lib/injector.js:48:43)
    at kotlin-frontend-plugin/examples/frontend-only/build/node_modules/karma/lib/server.js:166:20
    at Array.forEach (<anonymous>)
    at Server._start (kotlin-frontend-plugin/examples/frontend-only/build/node_modules/karma/lib/server.js:165:21)
    at Injector.invoke (kotlin-frontend-plugin/examples/frontend-only/build/node_modules/di/lib/injector.js:75:15)
    at Server.start (kotlin-frontend-plugin/examples/frontend-only/build/node_modules/karma/lib/server.js:126:18)
    at Object.exports.run (kotlin-frontend-plugin/examples/frontend-only/build/node_modules/karma/lib/cli.js:283:26)
    at Object.<anonymous> (kotlin-frontend-plugin/examples/frontend-only/build/node_modules/karma/bin/karma:3:23)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:501:12)


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':karma-run-single'.
> Process 'command '~/.gradle/nodejs/node-v9.9.0-linux-x64/bin/node'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
11 actionable tasks: 2 executed, 9 up-to-date
```

**Comment by plannigan** [comment]
Adding quint as a devDependency allows the example to run. So it seems like this is a documentation issue. The readme should explain that the testing framework should be listed as a dependency. Also, the examples should be updated so they run out of the box.

In general, the example seem like they could use a bit more love. There are a number of things commented out without an explanation. There aren't any comments to indicate why each of the configurations settings are being set, as apposed to the default, and why values are what they are.

**Comment by cy6erGn0m** [comment]
Fixed in 0.0.31
